### PR TITLE
Add new provider : Xing

### PR DESCRIPTION
**Pull request by D4rkTiger**
Add Xing provider to authenticate with german professional social
network account

### CODE REVIEW

**Comment by D4rkTiger** [comment]
I want to add a question. I develop a custom configuration section that allows to put credentials in web.config. Anyone wants that is useful to put it in this project or not ? The goal is to add all the others providers already registered in this project

**Comment by D4rkTiger** [comment]
Do I close this pull request to be merged in the project ? Is there a validation by members before ?

File: Owin.Security.Providers/packages.config [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.Net.Compilers" version="1.1.1" targetFramework="net45" developmentDependency="true" />
```
Comment (jerriep) [inline Owin.Security.Providers/packages.config]: Why is this package added?

File: Owin.Security.Providers/Owin.Security.Providers.csproj [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.Net.Compilers.1.1.1\build\Microsoft.Net.Compilers.props" Condition="Exists('..\packages\Microsoft.Net.Compilers.1.1.1\build\Microsoft.Net.Compilers.props')" />
```
Comment (jerriep) [inline Owin.Security.Providers/Owin.Security.Providers.csproj]: Why did you add this package?

File: Owin.Security.Providers/Owin.Security.Providers.csproj [hunk 3]
```diff
@@ -448,8 +464,16 @@
     <None Include="app.config" />
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="DoYouBuzz\" />
```
Comment (jerriep) [inline Owin.Security.Providers/Owin.Security.Providers.csproj]: Why is this empty folder added to the project?

File: Owin.Security.Providers/Owin.Security.Providers.csproj (hunk 2 above)
Comment (D4rkTiger) [inline Owin.Security.Providers/Owin.Security.Providers.csproj]: It is necessary to compile C# 6.0. Not for you ?

File: Owin.Security.Providers/packages.config (hunk 1 above)
Comment (D4rkTiger) [inline Owin.Security.Providers/packages.config]: It is necessary to compile C# 6.0. Not for you ?

File: Owin.Security.Providers/Owin.Security.Providers.csproj (hunk 3 above)
Comment (D4rkTiger) [inline Owin.Security.Providers/Owin.Security.Providers.csproj]: Oh sorry, it is a future provider I prepare to provide. I haven't seen that I checked-in

**Comment by D4rkTiger** [comment]
Sorry for my noob experience on GitHub, but for now what do I should do. I can correct and submit again or these steps will be done after ? It's because I can submit the other provider at the same time if you want and you can check both.

For the nuget-package, I must remove it or I can leave it in the project for developers who haven't Visual Studio 2015 ?

**Comment by jerriep** [comment]
Yes. Please remove the folder and commit the changes.

Also, please remove the C# 6 packages and C# 6 syntax from the code.I do not want to force people t use it.

Once you have done that you can commit the changes and push them up to GH again, and this PR will automatically update.

**Comment by D4rkTiger** [comment]
Ok, sorry for the delay, I was very occupied in my job. I do it, remove C# 6 code (VKontakte), the added nuget package, remove the folder "DoYouBuzz" and submit again.

I do my best to do this today :-)

**Comment by D4rkTiger** [comment]
Ok It's done ;-)
